src/cli.rs

```diff
@@ -154,79 +154,220 @@ pub struct Cli {
     #[clap(disable_version_flag = true)]
-    Suggest(SuggestArgs),
+    RecordViolation(RecordViolationArgs),
 
+    // Plumbing/Debug Commands
     /// Reformat all of vet's files (in case you hand-edited them)
     ///
-    /// All commands that access the store (supply-chain) will implicitly do this.
+    /// Most commands will implicitly do this, so this mostly exists as "plumbing"
+    /// for building tools on top of vet, or in case you don't want to run another command.
     #[clap(disable_version_flag = true)]
     Fmt(FmtArgs),
 
     /// Explicitly fetch the imports (foreign audit files)
     ///
-    /// `cargo vet check` will implicitly do this.
+    /// `cargo vet check` will implicitly do this, so this mostly exists as "plumbing"
+    /// for building tools on top of vet.
     #[clap(disable_version_flag = true)]
     FetchImports(FetchImportsArgs),
 
-    /// Regenerate the 'unaudited' entries to try to minimize them and make the vet pass
-    #[clap(disable_version_flag = true)]
-    RegenerateUnaudited(RegenerateUnauditedArgs),
-
-    /// Print a mermaid-js visualization of the cargo build graph as understood by cargo-vet
+    /// Print the cargo build graph as understood by `cargo vet`
+    ///
+    /// This is a debugging command, the output's format is not guaranteed.
+    /// Use `cargo metadata` to get a stable version of what *cargo* thinks the
+    /// build graph is. Our graph is based on that result.
+    ///
+    /// With `--output-format=human` (the default) this will print out mermaid-js
+    /// diagrams, which things like github natively support rendering of.
+    ///
+    /// With `--output-format=json` we will print out more raw statistics for you
+    /// to search/analyze.
+    ///
+    /// Most projects will have unreadably complex build graphs, so you may want to
+    /// use the global `--filter-graph` argument to narrow your focus on an interesting
+    /// subgraph. `--filter-graph` is applied *before* doing any semantic analysis,
+    /// so if you filter out a package and it was the problem, the problem will disappear.
+    /// This can be used to bisect a problem if you get ambitious enough with your filters.
     #[clap(disable_version_flag = true)]
     DumpGraph(DumpGraphArgs),
 
     /// Print --help as markdown (for generating docs)
+    ///
+    /// The output of this is not stable or guaranteed.
     #[clap(disable_version_flag = true)]
     #[clap(hide = true)]
     HelpMarkdown(HelpMarkdownArgs),
 
     /// Clean up old packages from the vet cache
     ///
-    /// Removes  packages which haven't been accessed in a while, and deletes
+    /// Removes packages which haven't been accessed in a while, and deletes
     /// any extra files which aren't recognized by cargo-vet.
     ///
     /// In the future, many cargo-vet subcommands will implicitly do this.
     #[clap(disable_version_flag = true)]
     Gc(GcArgs),
 }
 
+#[derive(Subcommand)]
+pub enum RegenerateSubcommands {
+    ///
```

Comment:
This should probably have a non-empty doc comment

```suggestion
    /// Regenerate the `exemptions` list, removing unnecessary crates, and
    /// adding new ones such that `cargo vet check` will pass.
```